src/coordination/coordinator_instance.cpp

```diff
@@ -121,26 +140,36 @@ auto CoordinatorInstance::ShowInstances() const -> std::vector<InstanceStatus> {
       std::ranges::transform(repl_instances_, std::back_inserter(instances_status), process_repl_instance_as_leader);
     }
   } else {
-    auto const stringify_inst_status = [](ReplicationRole status) -> std::string {
-      return status == ReplicationRole::MAIN ? "main" : "replica";
+    auto const stringify_inst_status = [raft_state_ptr = &raft_state_](
```

Comment:
Is this used elsewhere this pointer capturing? I don't have anything against just asking

Reply:
I am not sure to be honest